gooddata-sdk/gooddata_sdk/catalog/declarative/analytics_layer.py

```diff
@@ -0,0 +1,155 @@
+# (C) 2022 GoodData Corporation
```

Comment:
Refactor classes into directory structure representing the hierarchy of metadata objects:

declarative -> workspace -> analyticsModel -> dashboards (insights, metrics)
declarative -> workspace -> analyticsModel -> insights
declarative -> workspace -> analyticsModel -> metrics
declarative -> workspace -> logicalModel -> datasets -> attribute + label, facts
declarative -> workspace -> logicalModel -> dateDatasets